Tools/Scripts/libraries/webkitcorepy/webkitcorepy/string_utils.py

```diff
@@ -92,23 +92,31 @@ def out_of(number, base):
     return '[{}{}/{}]'.format(' ' * (len(base) - len(number)), number, base)
 
 
-def elapsed(seconds):
-    if seconds <= 0:
-        return 'no time'
-    elif seconds < 1:
-        return 'less than a second'
-
-    seconds = int(round(seconds))
+def elapsed(seconds: float, precision: int = None, shorthand: bool = False):
+    rounded_seconds = round(seconds, ndigits=precision)
+    if seconds < 60:
```

Comment:
Given the output here is `rounded_seconds`, shouldn't this be `rounded_seconds < 60`?

Testing `elapsed(59.9)` and other values around boundaries would probably be good (likewise `60 * 60 - 0.1` and `60 * 60 * 24 - 0.1`).